plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterAttribute.php

```diff
@@ -166,19 +166,25 @@ protected function render( $block_attributes, $content, $block ) {
 		$product_attribute = wc_get_attribute( $block_attributes['attributeId'] );
 		$attribute_counts  = $this->get_attribute_counts( $block, $product_attribute->slug, $block_attributes['queryType'] );
 		$hide_empty        = $block_attributes['hideEmpty'] ?? true;
+		$orderby           = $block_attributes['sortOrder'] ? explode( '-', $block_attributes['sortOrder'] )[0] : 'name';
+		$order             = $block_attributes['sortOrder'] ? strtoupper( explode( '-', $block_attributes['sortOrder'] )[1] ) : 'DESC';
 
 		if ( $hide_empty ) {
 			$attribute_terms = get_terms(
 				array(
 					'taxonomy' => $product_attribute->slug,
 					'include'  => array_keys( $attribute_counts ),
+					'orderby'  => $orderby,
+					'order'    => $order,
 				)
 			);
 		} else {
 			$attribute_terms = get_terms(
 				array(
 					'taxonomy'   => $product_attribute->slug,
 					'hide_empty' => false,
+					'orderby'    => $orderby,
+					'order'      => $order,
 				)
 			);
 		}
```

Comment:
I know it's not in scope of this PR but I can't help but notice the duplication in this `if else` statement. Super nitpick but you could consider something like:

```suggestion
		$args = array(
			'taxonomy' => $product_attribute->slug,
			'orderby'  => $orderby,
			'order'    => $order,
		);

		if ( $hide_empty ) {
			$args['include'] = array_keys( $attribute_counts );
		} else {
			$args['hide_empty'] = false;
		}

		$attribute_terms = get_terms( $args );
```

Leaving it up to you to decide.